.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         run: flutter format lib test -l 80 --set-exit-if-changed
 
       - name: analyzer
-        run: flutter analyze --fatal-warnings --fatal-infos .
+        run: flutter analyze --fatal-warnings --fatal-infos ./lib ./test/
```

Comment:
Added this, because analyzer check was failing in the example directory for flutter v2.8.0 with an issue that seemingly is not so significant.
https://github.com/supabase/supabase-flutter/actions/runs/4261285326/jobs/7415418170

Reply:
Hmm, still failing...